### PR TITLE
Advanced Global CDN

### PR DESCRIPTION
**Pull request by alexfornuto**
Closes DOC-68

## Effect
PR includes the following changes:
- New doc on the Advanced Global CDN offering from PS.

## Remaining Work
- [x] Finish Writing
- [x] Technical review
- [x] Copy Review
- [x] Add to landing page(s)
- [x] Proper tags/category
- [x] Consider adding @tmountjr as a contributor to this doc.

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] Include pages within docs search service provider
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5538 | Difference |
| --- | --- | --- | --- |
| Performance Score | `57.99999999999999` | `54` | `-4` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5538` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18643/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18643/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5538 | Difference |
| --- | --- | --- | --- |
| Performance Score | `57.99999999999999` | `52` | `-6` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5538` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18647/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18647/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5538 | Difference |
| --- | --- | --- | --- |
| Performance Score | `54` | `51` | `-3` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5538` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18652/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18652/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5538 | Difference |
| --- | --- | --- | --- |
| Performance Score | `54` | `54` | `0` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5538` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18663/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18663/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5538 | Difference |
| --- | --- | --- | --- |
| Performance Score | `55.00000000000001` | `53` | `-2` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5538` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18667/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18667/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

File: source/data/contributor.yaml [hunk 1]
```diff
@@ -347,3 +347,8 @@
   linkedin: //www.linkedin.com/in/aleksandr-korolev-575045148
   drupal: //www.drupal.org/u/aleksandrkorolyov
   bio: Software Engineer | DrupalSquad
+- id: tmountjr
+  name: Tom Mount
+  avatar: //avatars.githubusercontent.com/u/1203675
+  github: //github.com/tmountjr
+  bio: Sr. Technical Engagement Manager
```
Comment (alexfornuto) [inline source/data/contributor.yaml]: @tmountjr feel free to adjust your ID, avatar, etc, and add other links if you want. Unfortunately this feature requires an externally hosted avatar image.

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5538 | Difference |
| --- | --- | --- | --- |
| Performance Score | `54` | `50` | `-4` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5538` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18671/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18671/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5538 | Difference |
| --- | --- | --- | --- |
| Performance Score | `57.99999999999999` | `43` | `-15` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5538` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18675/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18675/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5538 | Difference |
| --- | --- | --- | --- |
| Performance Score | `50` | `41` | `-9` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5538` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18679/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18679/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5538 | Difference |
| --- | --- | --- | --- |
| Performance Score | `51` | `56.99999999999999` | `6` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5538` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18718/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18718/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5538 | Difference |
| --- | --- | --- | --- |
| Performance Score | `59` | `56.99999999999999` | `-2` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5538` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18728/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18728/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5538 | Difference |
| --- | --- | --- | --- |
| Performance Score | `54` | `44` | `-10` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5538` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18735/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18735/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5538 | Difference |
| --- | --- | --- | --- |
| Performance Score | `55.00000000000001` | `54` | `-1` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5538` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18742/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18742/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5538 | Difference |
| --- | --- | --- | --- |
| Performance Score | `59` | `57.99999999999999` | `-1` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5538` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18743/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18743/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5538 | Difference |
| --- | --- | --- | --- |
| Performance Score | `56.00000000000001` | `54` | `-2` |
| Best Practice Score | `79` | `79` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5538` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18754/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18754/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5538 | Difference |
| --- | --- | --- | --- |
| Performance Score | `60` | `55.00000000000001` | `-5` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5538` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18755/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18755/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5538 | Difference |
| --- | --- | --- | --- |
| Performance Score | `50` | `46` | `-4` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5538` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18758/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18758/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Pull request merged**